plugins/CivilTongueEx/class.civiltongueex.plugin.php

```diff
@@ -75,14 +75,14 @@ public function profileController_render_before($Sender, $Args) {
      * @param array $Args
      */
     public function activityController_render_before($Sender, $Args) {
-        $User = val('User', $Sender);
+      $User = GetValue('User', $Sender);
         if ($User)
-            setValue('About', $User, $this->replace(val('About', $User)));
+         SetValue('About', $User, $this->Replace(GetValue('About', $User)));
 
         if (isset($Sender->Data['Activities'])) {
             $Activities =& $Sender->Data['Activities'];
             foreach ($Activities as &$Row) {
-                setValue('Story', $Row, $this->replace(val('Story', $Row)));
+            SetValue('Story', $Row, $this->Replace(GetValue('Story', $Row)));
```

Comment:
This sort of thing indicates a merge conflict wasn't resolved correctly - coding standards regression.